packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -2,55 +2,118 @@ import _ from 'intl'
+      }),
+      closeSupportTunnel: async () => ({
+        isSupportTunnelClosed: await closeSupportTunnel(),
+      }),
     },
   }),
   injectState,
-  ({ effects, state: { stdoutCheckXoa } }) => (
+  ({
+    effects,
+    state: { stdoutCheckXoa, stdoutOpenSupportTunnel, isSupportTunnelClosed },
+  }) => (
     <Container>
+      {COMMUNITY && (
+        <Row className='mb-2'>
+          <Col>
+            <span className='text-info'>{_('supportCommunity')}</span>
+          </Col>
+        </Row>
+      )}
       <Row>
         <Col mediumSize={6}>
           <Card>
             <CardHeader>{_('xoaCheck')}</CardHeader>
-            {COMMUNITY ? (
-              <CardBlock>
-                <span className='text-info'>{_('checkXoaCommunity')}</span>
-              </CardBlock>
-            ) : (
-              <CardBlock>
-                <ActionButton
-                  btnStyle='success'
-                  handler={effects.checkXoa}
-                  icon='diagnosis'
-                >
-                  {_('checkXoa')}
-                </ActionButton>
-                <hr />
+            <CardBlock>
+              <ActionButton
+                btnStyle='success'
+                disabled={COMMUNITY}
+                handler={effects.checkXoa}
+                icon='diagnosis'
+              >
+                {_('checkXoa')}
+              </ActionButton>
+              <hr />
+              <pre
+                dangerouslySetInnerHTML={{
+                  __html: ansiUp.ansi_to_html(stdoutCheckXoa),
+                }}
+              />
+            </CardBlock>
+          </Card>
+        </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('supportTunnel')}</CardHeader>
+            <CardBlock>
+              <Row>
+                <Col mediumSize={6}>
+                  <ActionButton
+                    btnStyle='success'
+                    disabled={COMMUNITY || !isSupportTunnelClosed}
+                    handler={effects.openSupportTunnel}
+                    icon='shown'
+                  >
+                    {_('openTunnel')}
+                  </ActionButton>
+                </Col>
+                <Col mediumSize={6}>
+                  <ActionButton
+                    disabled={COMMUNITY || isSupportTunnelClosed}
+                    btnStyle='primary'
+                    className='pull-right'
+                    handler={effects.closeSupportTunnel}
+                    icon='hidden'
+                  >
+                    {_('closeTunnel')}
+                  </ActionButton>
+                </Col>
+              </Row>
+              <hr />
+              {isSupportTunnelClosed ? (
+                <span>{_('supportTunnelClosed')}</span>
+              ) : (
                 <pre
                   dangerouslySetInnerHTML={{
-                    __html: ansiUp.ansi_to_html(stdoutCheckXoa),
+                    __html: marked(stdoutOpenSupportTunnel),
```

Comment:
This should be a computed, no need to parse at each render (we should have done the same for `stdoutXoaCheck`, maybe in another PR).